packages/frontend/editor-ui/src/features/dataStore/components/dataGrid/DataStoreTable.vue

```diff
@@ -424,6 +426,18 @@ const onAddRowClick = async () => {
 	}
 };
 
+// Reusable date formatter function
+const formatDateValue = (value: any): string => {
+	if (!value) return '';
```

Comment:
I'd suggest to check what the actual type of the value is here and simplify the function to only handle that type.